### PR TITLE
Update docstrings for BSgate and MZgate to clarify convention

### PR DESCRIPTION
**Pull request by elib20**
**Context:** To clarify the BSgate and MZgate conventions in the docstrings

**Description of the Change:** Added info in the docstrings

**Benefits:** Better documentation

**Possible Drawbacks:** None

**Related GitHub Issues:**

### CODE REVIEW

File: mrmustard/lab/gates.py [hunk 1]
```diff
@@ -335,6 +335,9 @@ class BSgate(Parametrized, Transformation):
     It applies to a single pair of modes.
     One can optionally set bounds for each parameter, which the optimizer will respect.
 
+    N.B. the phase in BSgate(theta,phi,modes=[i,j]) is equivalent to
```
Comment (sduquemesa) [inline mrmustard/lab/gates.py:338]: What does "N.B." stands for?

File: mrmustard/lab/gates.py [hunk 2]
```diff
@@ -335,6 +335,9 @@ class BSgate(Parametrized, Transformation):
...
+    Rgate(-phi, modes=[j]) >> BSgate(theta) >> Rgate(phi, modes=[j])
+
```
Comment (sduquemesa) [inline mrmustard/lab/gates.py:340]: Use sphinx syntax to format code
```suggestion
    N.B. the phase in ``BSgate(theta,phi,modes=[i,j])`` is equivalent to
    ``Rgate(-phi, modes=[j]) >> BSgate(theta) >> Rgate(phi, modes=[j])``
```

**Review by sduquemesa — COMMENTED** [review]
Thanks @elib20!

File: mrmustard/lab/gates.py [hunk 3]
```diff
@@ -383,15 +386,20 @@ class MZgate(Parametrized, Transformation):
 
     It supports two conventions:
         1. if ``internal=True``, both phases act inside the interferometer: ``phi_a`` on the upper arm, ``phi_b`` on the lower arm;
-        2. if ``internal = False``, both phases act on the upper arm: ``phi_a`` before the first BS, ``phi_b`` after the first BS.
+        2. if ``internal=False``, both phases act on the upper arm: ``phi_a`` before the first BS, ``phi_b`` after the first BS.
 
     One can optionally set bounds for each parameter, which the optimizer will respect.
 
+    N.B. when internal=False, MZgate is equivalent to Rgate(phi_a) >> BSgate(pi/4, pi/2) >> Rgate(phi_b) >> BSgate(-pi/4, -pi/2),
+    and when internal=True, MZgate is equivalent to BSgate(pi/4, pi/2)>>Rgate([phi_a, phi_b]) >> BSgate(-pi/4, -pi/2).
```
Comment (elib20) [inline mrmustard/lab/gates.py:394]:
```suggestion
    N.B. when internal=False, MZgate is equivalent to ``Rgate(phi_a) >> BSgate(pi/4, pi/2) >> Rgate(phi_b) >> BSgate(-pi/4, -pi/2)``,
    and when internal=True, MZgate is equivalent to ``BSgate(pi/4, pi/2)>>Rgate([phi_a, phi_b]) >> BSgate(-pi/4, -pi/2)``.
```

File: mrmustard/lab/gates.py [hunk 4]
```diff
@@ -335,6 +335,8 @@ class BSgate(Parametrized, Transformation):
     It applies to a single pair of modes.
     One can optionally set bounds for each parameter, which the optimizer will respect.
 
+    N.B. the phase in ``BSgate(theta,phi,modes=[i,j])`` is equivalent to
```
Comment (elib20) [inline mrmustard/lab/gates.py:338]:
```suggestion
    The phase in ``BSgate(theta,phi,modes=[i,j])`` is equivalent to
```

File: mrmustard/lab/gates.py [hunk 5]
```diff
@@ -383,15 +385,20 @@ class MZgate(Parametrized, Transformation):
 
     It supports two conventions:
         1. if ``internal=True``, both phases act inside the interferometer: ``phi_a`` on the upper arm, ``phi_b`` on the lower arm;
-        2. if ``internal = False``, both phases act on the upper arm: ``phi_a`` before the first BS, ``phi_b`` after the first BS.
+        2. if ``internal=False``, both phases act on the upper arm: ``phi_a`` before the first BS, ``phi_b`` after the first BS.
 
     One can optionally set bounds for each parameter, which the optimizer will respect.
 
+    N.B. when internal=False, MZgate is equivalent to ``Rgate(phi_a) >> BSgate(pi/4, pi/2) >> Rgate(phi_b) >> BSgate(-pi/4, -pi/2)``,
```
Comment (elib20) [inline mrmustard/lab/gates.py:392]:
```suggestion
    When internal=False, MZgate is equivalent to ``Rgate(phi_a) >> BSgate(pi/4, pi/2) >> Rgate(phi_b) >> BSgate(-pi/4, -pi/2)``,
```

File: .readthedocs.yml
Comment (ziofil) [inline .readthedocs.yml:1]: out of curiosity, what do these changes do?

**Pull request closed without merging**